src/commands.c

```diff
@@ -54,7 +54,7 @@ const struct _global_command_struct commands[] = {
 { .command="cat",       .allow_hiz=true,  .func=&disk_cat_handler,                   .help_text=0x00 }, // cat T_CMDLN_CAT
 { .command="m",         .allow_hiz=true,  .func=&ui_mode_enable_args,                .help_text=0x00 }, // "m"  T_CMDLN_MODE //needs trailing int32
 { .command="W",         .allow_hiz=false, .func=&psucmd_enable_handler,              .help_text=0x00 }, // "W"   T_CMDLN_PSU_EN  //TOD0: more flexability on help handling and also a general deescription
-{ .command="#",         .allow_hiz=true,  .func=&cmd_mcu_reset_handler,              .help_text=T_CMDLN_RESET }, // "#"

```

Comment:
This is what the command parser looks for.  Note, however, that this does NOT change the help screens.